02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,27 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
+
+  <title>My First Webpage</title>
   <link rel="stylesheet" href="css/style.css">
+
 </head>
+
 <body>
 
+   <h1>Goodbye, space!</h1>
```

Comment:
Thank you!  I started researching new elements to implement.Thanks again,  Rey   

    On Monday, May 9, 2016 4:59 PM, fasteddiegarcia <notifications@github.com> wrote:
 

 In 02LessonTwo/homework/index.html:>  <body>
>  
> +   <h1>Goodbye, space!</h1>
I like the indentation and use of many elements. You seem to have a grasp of this, so I'd like to see more. Try to use more elements like , , 
| ,    
,    
   - . Great beginning!
 |

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub